src/app/gui/object/object.directive.ts

```diff
@@ -0,0 +1,30 @@
+import { Directive, Input } from "@angular/core";
+import { ObjectComponent } from "./object.component";
+import { LuJsonService, LuLocaleService } from "../../services";
+
+@Directive({
+  selector: "lux-object[luxFetch]"
+})
+export class ObjectDirective {
+
+  @Input("luxFetch") set id(id: number) {
+    this.objectComponent.id = id;
+    this.objectComponent.title = `#${id}`;
+    this.luJson.getObject(id).subscribe(this.onObject);
+  }
+
+  constructor(private luJson: LuJsonService, private luLocale: LuLocaleService, private objectComponent: ObjectComponent) { }
+
+  onObject = (object: any) => {
```

Comment:
```suggestion
  onObject(object: any) {
```

Reply:
Ah yeah, that was an early solution to get methods to work properly as callbacks. I've found the `.bind` solution in the meantime which doesn't require changing the method signature, I'll be changing it to use that instead.